python/send_message/README.md

```diff
@@ -0,0 +1,119 @@
+# SendMessage()
+
+A Python Cloud Function for sending a message using a specific channel to a receiver
+
+Supported channels are `SMS`, `Email` ,`Disocrd` and `Twitter`.
+
+_SMS Example payload_
+
+```json
+{ "type": "SMS", "receiver": "+123456789", "message": "Programming is fun!" }
+```
+
+_Email Example payload_
+
+```json
+{
+  "type": "Email",
+  "receiver": "hello@example.com",
+  "message": "Programming is fun!",
+  "subject": "Programming is funny!"
+}
+```
+
+_Discord Example payload_
+
+```json
+{
+  "type": "Discord",
+  "message": "Hi"
+}
+```
+
+_Twitter Example payload_
+
+```json
+{
+  "type": "Twitter",
+  "receiver": "",
+  "message": "Programming is fun!"
+}
+```
+
+_Successful function response:_
+
+```json
+{
+  "success": true
+}
+```
+
+_Error function response:_
+
+```json
+{
+  "success": false,
+  "error": "Failed to send message,check webhook URL"
```

Comment:
Would you please make sure the key is `message` rather than `error` in all error responses to match the issue requirements?